frontend/src/lib/components/Errors/types.ts

```diff
@@ -114,6 +114,20 @@ export interface ExceptionAttributes {
     level?: string
     url?: string
     handled?: boolean
+    exceptionReleases?: RawEventExceptionRelease[]
+}
+
+export interface RawEventExceptionRelease {
```

Comment:
Why the `Raw`? I think `EventExceptionRelease` is clear enough here

```suggestion
export interface EventExceptionRelease {
```

Reply:
Oops. Forgot to change that. Done